packages/radix-ui-themes/src/components/select.tsx

```diff
@@ -126,7 +126,7 @@ const SelectContent = React.forwardRef<SelectContentElement, SelectContentProps>
                 </ScrollAreaPrimitive.Viewport>
               </SelectPrimitive.Viewport>
               <ScrollAreaPrimitive.Scrollbar
-                className="rt-ScrollAreaScrollbar size-1"
+                className="rt-ScrollAreaScrollbar rt-r-size-1"
```

Comment:
It also changes the width somewhat abruptly when disappears, but that's a fix for another time I guess